### PR TITLE
FISH-9854 Fix high priority spotbugs errors in common-util

### PR DESCRIPTION
**Pull request by kalinchan**
## Description

Various spotbugs fixes

Mainly setting default charset and adding ProcessState instead of boolean.

Also deprecated methods to keep semantic versioning happy 
### New tests

n/a
### Testing Performed

Started server
## Documentation

n/a
## Notes for Reviewers

Ran spotbugs in common-util no more high priority warnings

### CODE REVIEW

**Review by breakponchito — APPROVED** [review]
LGTM

File: nucleus/common/common-util/src/main/java/com/sun/enterprise/universal/process/ProcessUtils.java [hunk 1]
```diff
@@ -179,15 +190,15 @@ public static String killJvm(String classname) {
      * @return true if it's running, false if not and null if we don't know. I.e
      * the return value is a true tri-state Boolean.
      */
-    public static Boolean isProcessRunning(int aPid) {
+    public static ProcessState isProcessRunningState(int aPid) {
```
Comment (svendiedrichsen) [inline nucleus/common/common-util/src/main/java/com/sun/enterprise/universal/process/ProcessUtils.java:193]: Method name sounds strange now that this is actually returning some state enum. Shouldn't it be get...?

File: nucleus/common/common-util/src/main/java/com/sun/enterprise/universal/process/ProcessUtils.java [hunk 2]
```diff
@@ -197,7 +208,7 @@ public static Boolean isProcessRunning(int aPid) {
     private static final String[] paths;
    private static boolean debug;
 
-   private static boolean isProcessRunningWindows(int aPid) throws ProcessManagerException {
+   private static ProcessState isProcessRunningWindows(int aPid) throws ProcessManagerException {
```
Comment (svendiedrichsen) [inline nucleus/common/common-util/src/main/java/com/sun/enterprise/universal/process/ProcessUtils.java:211]: Shouldn't it be get...?

File: nucleus/common/common-util/src/main/java/com/sun/enterprise/universal/process/ProcessUtils.java [hunk 3]
```diff
@@ -222,20 +233,20 @@ private static boolean isProcessRunningWindows(int aPid) throws ProcessManagerEx
...
             else
-                return false;
+                return ProcessState.STOPPED;
         }
 
         throw new ProcessManagerException("unknown");
     }
 
-    private static Boolean isProcessRunningUnix(int aPid) throws ProcessManagerException {
+    private static ProcessState isProcessRunningUnix(int aPid) throws ProcessManagerException {
```
Comment (svendiedrichsen) [inline nucleus/common/common-util/src/main/java/com/sun/enterprise/universal/process/ProcessUtils.java:244]: Shouldn't it be get...?

File: nucleus/common/common-util/src/main/java/com/sun/enterprise/util/io/ServerDirs.java [hunk 4]
```diff
@@ -127,24 +129,13 @@ public ServerDirs(File leaf) throws IOException {
...
 
         String localPasswordBuffer = null;  // need an atomic assign tor localPassword
-        BufferedReader r = null;
-        try {
-            r = new BufferedReader(new FileReader(localPasswordFile));
+        try (BufferedReader r = new BufferedReader(new FileReader(localPasswordFile, StandardCharsets.UTF_8))) {
             localPasswordBuffer = r.readLine();
-        }
-        catch (Exception e) {
+        } catch (Exception e) {
```
Comment (svendiedrichsen) [inline nucleus/common/common-util/src/main/java/com/sun/enterprise/util/io/ServerDirs.java:134]: How about logging ?

**Review by svendiedrichsen — APPROVED** [review]
Looks good for me. Thank you.

**Pull request merged**